docs/developer-docs/smart-contracts/maintain/delete.mdx

```diff
@@ -5,45 +5,15 @@ keywords: [beginner, tutorial, maintain canisters, delete canisters, delete]
 import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
 
-
 # Delete
 
-<MarkdownChipRow labels={["Beginner", "Tutorial"]} />
+<MarkdownChipRow labels={["Beginner", "Concept"]} />
 
 ## Overview
 
-If you want to permanently delete a specific <GlossaryTooltip>canister</GlossaryTooltip> or all canisters for a specific project on a given deployment (either local or remote), you can do so by running the command:
-
-```
-dfx canister delete <canister-name> // Delete local canister
-dfx canister delete <canister-name> --network ic // Delete canister on mainnet
-```
-
-Deleting a canister removes the canister identifier, code, and state. Before you can delete a canister, however, you must first stop the canister to clear any pending message requests or replies.
-
-## Delete all canisters
-
-To delete all canisters for a project:
-
-- #### Step 1:  Open a new terminal and navigate to your project directory.
-
-- #### Step 2:  Start the local canister execution environment, if necessary.
-
-In most cases, this step is only necessary if you are running the canisters locally.
-
-If you were deleting canisters to run on a remote execution environment, e.g. ICP blockchain, you would include the `--network` command-line option to perform tasks on the environment specified under this parameter.
-
-- #### Step 3:  Check the status of the project canisters running on the local canister execution environment by running the following command:
-
-dfx canister status --all
-
-- #### Step 4:  Stop all of the project canisters by running the following command:
-
-dfx canister stop --all
-
-- #### Step 5:  Delete all of the project canisters by running the following command:
+If you want to permanently delete a specific <GlossaryTooltip>canister</GlossaryTooltip> or all canisters for a project, you can use the [`dfx canister delete <canister-name>`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-delete) command.
 
-dfx canister delete --all
+Deleting a canister removes the canister identifier, code, and state. Before you can delete a canister, you must first stop the canister using [`dfx canister stop <canister-name>`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-stop) to clear any pending message requests or replies.
```

Comment:
```suggestion
Deleting a canister permanently removes its code and state. This action cannot be undone and once the canister is deleted, its ID cannot be reused. Before you can delete a canister, you must first stop the canister using [`dfx canister stop <canister-name>`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-stop) to clear any pending message requests or replies.
```

Reply:
@jessiemongeon1 I am also wondering if we should add extra info about cycles owned by the canister. according to the IC spec those are discarded. is that correct? 🤔 

see https://internetcomputer.org/docs/current/references/ic-interface-spec#ic-delete_canister

Reply:
The cycles are returned to the controller's cycles balance, so they are not "discarded" in the sense they are burned. I think the spec wording for this is a bit misleading.